_posts/2021-06-30-newsletter.md

```diff
@@ -0,0 +1,75 @@
+
+
+### Origami Build Service version 2 is deprecated
+
+This version of the build service uses the now deprecated FT Bower Registry. We have built a new version, version 3, which does not use the FT Bower Registry. To help projects migrate to version 3, we have built [a tool](https://www.ft.com/__origami/service/build/url-updater) which will list out the specific steps for a project to do in order to successfully migrate over to the new version.
+
+We are tracking the projects which will need to be migrated to the new version in [the same spreadshet we are tracking the Bower migration](https://docs.google.com/spreadsheets/d/1Pem5e6cR0aiuKpYa7VD08AnSSynzjRtWt_VAHAoyhPQ/edit#gid=0).
+
+
+## Special thanks
+
+This issues special thanks goes to Xuan Zhang! Xuan worked on the design and interactions for the new o-autocomplete component and was very helpful in explaining the designs and how they should look on different devices.
+
+
+## Broader update
+
+- MAJOR: All Origami maintained components have been migrated from Bower to NPM
+- MAJOR: [origami-build-tools](https://github.com/Financial-Times/origami-build-tools) version 11 released, dropping support for Bower and adding fully support for NPM
+- MINOR: [o-colors](https://github.com/Financial-Times/o-colors) added a new palette named "partner content" with a new colour named "Matisse"
+- PATCH: [o-layout](https://github.com/Financial-Times/o-layout) added missing closing div to the footer markup in the component's demos
+- PATCH: [o-teaser](https://github.com/Financial-Times/o-teaser) updated the partner content label colour to "matisse"
+- DEPRECATED: [o-layers](https://github.com/Financial-Times/o-layers) was deprecated
+- PATCH:[o-tracking](https://github.com/Financial-Times/o-tracking) has added back the `source_id` property
+- The Origami blog now has [an RSS feed](https://origami.ft.com/feed.xml).
+- [Origami Build Service](https://github.com/Financial-Times/origami-build-service) version 3 was released along with a page to help projects update their URLs to use the latest version of Origami
+- The following projects have had additional updates to support the migration from Bower to NPM:
+    - [Polyfill Service](https://github.com/Financial-Times/polyfill-service) has migrated to using Origami components via NPM
+    - [Origami Build Service](https://github.com/Financial-Times/origami-build-service) has migrated to using Origami components via NPM
+    - [Origami Repo Data](https://github.com/Financial-Times/origami-repo-data) has migrated to using Origami components via NPM
+    - [Origami Workshop](https://github.com/Financial-Times/origami-workshop) has migrated to using Origami components via NPM
+    - [Create Origami Component](https://github.com/Financial-Times/create-origami-component) has been updated to create an Origami component which publishes to NPM
```

Comment:
This hasn't been released yet, the PR for it is --> https://github.com/Financial-Times/create-origami-component/pull/238

Reply:
Released (thanks for the review chee)